lib/utils/show_settings_dialog.dart

```diff
@@ -857,16 +870,17 @@ class SettingsDialogState extends ConsumerState<SettingsDialog> {
 
                             ''',
                             child: ElevatedButton(
-                              onPressed: () {
-                                ref.invalidate(partitionTrainProvider);
-                                ref
-                                    .read(partitionValidProvider.notifier)
-                                    .state = 15;
-                                ref.read(partitionTestProvider.notifier).state =
-                                    15;
+                              onPressed: () async {
                                 ref.invalidate(
```

Comment:
After a reset this seems to go to false, but then I exit the session and come back and it is still true?